src/pages/friend/FriendDraftPage.tsx

```
@@ -8,6 +8,10 @@ import { BsQuestionCircle } from 'react-icons/bs';
 import LetterTextBox from '@/components/letters/LetterTextBox';
 import DailyQuestionBox from '@/components/letters/DailyQuestionBox';
 
+import { BsQuestionCircle } from 'react-icons/bs';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'BsQuestionCircle'. Consider to delete it or rename it.

'BsQuestionCircle' is defined here:


(lint/suspicious/noRedeclare)